apiserver/facades/client/application/application.go

```diff
@@ -1565,10 +1565,14 @@ func (api *APIBase) DestroyConsumedApplications(args params.DestroyConsumedAppli
 			results[i].Error = common.ServerError(err)
 			continue
 		}
-		// TODO (anastasiamac 2019-03-29) This may need to be forced too.
-		// see https://bugs.launchpad.net/juju/+bug/1822050
-		err = app.Destroy()
+
+		op := app.DestroyOperation(arg.Force)
+		if arg.Force {
+			op.MaxWait = common.MaxWait(arg.MaxWait)
+		}
+		err = api.backend.ApplyOperation(op)
 		if err != nil {
+			logger.Warningf("operational error encountered destroying application %v: %v", appTag.Id(), op.Errors)
```

Comment:
This should b printed in a separate if-, if there are any :)

Reply:
Done. Could you check that this is what you intended?

Reply:
Yes, thank you! But could we please add to the message "consumed application" as in "destroying consumed application" otherwise it'll be indistinguishable from  normal application destroy.